test/create-article.ts

```diff
@@ -1,14 +1,26 @@
+import { namedNode } from '@rdfjs/data-model';
 import deepFilter from 'deep-filter';
+import { Options } from 'jsonld';
 import { JsonLdObj } from 'jsonld/jsonld-spec';
-import { BlankNode } from 'rdf-js';
+import { BlankNode, NamedNode } from 'rdf-js';
 import { termToString } from 'rdf-string';
 
 const isNotUndefined = (arg: unknown): boolean => arg !== undefined;
 
-export default (id?: BlankNode, name = id ? `Article ${id}` : 'Article'): JsonLdObj => (
+type Options = {
+  id?: BlankNode;
```

Comment:
I guess it doesn't make things easier to pass in a `string` `id`, as we need the `BlankNode` in the other calls to `set(...)` anyway.